app/Http/Controllers/User/HighlightsController.php

```diff
@@ -112,6 +113,10 @@ public function index(Request $request, $user_id)
         $sort_dir   = checkParam('sort_dir') ?? 'asc';
         $query      = checkParam('query');
 
+        if (!in_array(Str::lower($sort_dir), ['asc', 'desc'])) {
+            $sort_dir = 'desc';
+        }
+
```

Comment:
Related with the issue:

```log
Exception 'InvalidArgumentException' with message 'Order direction must be "asc" or "desc".' in /var/app/current/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:1984
```

`sort_dir` parameter given may be an incorrect value so, we should make sure that if the value is not equal to `asc` or `desc` it should take a default value to avoid the database throws an error because the orderBy operator is not correct.